format/spec.md

```diff
@@ -177,8 +177,10 @@ A **`map`** is a collection of key-value pairs with a key type and a value type.
 | **`decimal(P,S)`** | Fixed-point decimal; precision P, scale S                                | Scale is fixed [1], precision must be 38 or less |
 | **`date`**         | Calendar date without timezone or time                                   |                                                  |
 | **`time`**         | Time of day without date, timezone                                       | Microsecond precision [2]                        |
-| **`timestamp`**    | Timestamp without timezone                                               | Microsecond precision [2]                        |
-| **`timestamptz`**  | Timestamp with timezone                                                  | Stored as UTC [2]                                |
+| **`timestamp`**    | Timestamp, microsecond precision, without timezone                       | [2]                                              |
+| **`timestamptz`**  | Timestamp, microsecond precision, with timezone                          | [2]                                              |
+| **`timestamp_ns`** | Timestamp, nanosecond precision, without timezone                        | [2]                                              | [v3](#version-3) |
+| **`timestamptz_ns`** | Timestamp, nanosecond precision, with timezone                         | [2]                                              | [v3](#version-3) |
```

Comment:
I don't think this is clear enough. I think the version column should come first and be "Valid from" or similar. "Iceberg version" could be confused with the library version and we need to make it clear what this version means: that you are not allowed to use it in tables until that version.

Reply:
PTAL. I could also fill in the other rows with v1.